spring-cloud-gcp-secretmanager/src/test/java/org/springframework/cloud/gcp/secretmanager/it/SecretManagerPropertySourceIntegrationTests.java

```diff
@@ -101,28 +99,14 @@ public void testConfiguration() {
 		assertThat(byteArraySecret).isEqualTo("the secret data.".getBytes());
 	}
 
-	@Test
-	public void testConfigurationDisabled() {
```

Comment:
We're losing this test now.

Reply:
Gotcha - I moved this test to the configuration unit tests you wrote: SecretManagerBootstrapConfigurationTests